AdobeStockImage/etc/Model/Asset.php

```diff
@@ -0,0 +1,421 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model;
+
+use Magento\AdobeStockImage\Api\Data\AssetInterface;
+use Magento\AdobeStockImage\Model\ResourceModel\Asset as ResourceModel;
+use Magento\Framework\Model\AbstractModel;
+
+class Asset extends AbstractModel implements AssetInterface
```

Comment:
I think this file needs to be removed from the pull request

Reply:
Deleted